### PR TITLE
Fixed last selected page after search

### LINKED ISSUE #2296: Backend search - save information of clicked search element for tree view
| Q | A |
| --- | --- |
| Bug? | no |
| New Feature? | yes |
| Sulu Version | 1.2 (and all before) |
| Browser Version | all |
#### Actual Behavior

When searching for a page in the Sulu-Backend-Search and clicking on one of the result pages, the page will be correctly loaded.

On that loaded page there is at the top left corner a back button - pressing that back button opens the tree view of all pages but without selecting the current page. That means I don't know where the page is located inside the tree.
#### Expected Behavior

When clicking on a page outgoing from the Sulu-Backend-Search, some informations should be saved. After clicking the page and then the top left back button, the node tree should have the information which page was just selected.
#### Steps to Reproduce
1. Login to Sulu, use Search and search for a page that exists
2. Click on the search result to open the page
3. Click on the back button

### PR DESCRIPTION
**Pull request by ReneZumtobel**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Fixed tickets | fixes #2296 |
| Related issues/PRs | #issuenum |
| License | MIT |
| Documentation PR | sulu/sulu-docs#prnum |
#### What's in this PR?

When opening a page over the search tab the last selected page is now changed.

### CODE REVIEW

File: src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/column/main.js [hunk 1]
```diff
@@ -213,8 +213,7 @@ define([
                     // if no action icon is rendered the data should not be loaded
                     return;
                 }
-
-                this.setLastSelected(item.id);
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/column/main.js]: is this function called anywhere else or can it be removed?

File: src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js [hunk 2]
```diff
@@ -756,6 +756,9 @@ define([
                         '/' + this.options.language + '/edit:' + this.data.id + '/settings'
                     );
                 }
+                
+                // set current page as last selected one
+                this.sandbox.sulu.saveUserSetting(this.options.webspace + 'ColumnNavigationSelected', this.options.id);
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js]: i would capsule this into a function.

File: src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/column/main.js (hunk 1 above)
Comment (ReneZumtobel) [inline src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/column/main.js]: Yeah it is still used somewhere else.

Comment (ReneZumtobel) [inline src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/column/main.js]: yeah it is sill used in other parts of the code

**Pull request merged**